tor_ocr/main.py

```diff
@@ -136,8 +181,11 @@ def run(config):
 
     tor_post = config.r.submission(id=clean_id(tor_post_id))
 
-    thing_to_reply_to = tor_post.reply(_(base_comment))
-    for chunk in chunks(result, 9000):
+    thing_to_reply_to = tor_post.reply(
+        _(base_comment.format(result['process_time_in_ms'] / 1000))
+    )
+
+    for chunk in chunks(result['text'], 9000):
```

Comment:
Not pertinent to this PR, but I'd like to eventually see us rework the 9000-character thing here into something a little more invisible. Seems like it's potentially a lot of duplication that we could pull out of every instance where we reply to comments.

Reply:
Really? I thought it was a rather elegant way of solving that issue. There's probably a built in function for it though :P

Reply:
Definitely -- I kind of just threw this in as a "ehhh that'll work".

Reply:
Honestly, this is about as elegant as it can get until we write a wrapper function around the PRAW model’s reply method. Not bad, but the moment something changes—say we decide to add a footer to all comments, not just the last one—we would have to update the 9000 in all of those spots.

Just seemed like logic that would be used in several spots that we could extract into another method. 🤷‍♂️

(Again, a bit off-topic for this PR, but still wanted to point it out)

Reply:
Yeah that makes sense.

(Also it does have a footer to all comments, that's what the aptly named function `_` does.)